BB-SC/MWLayer/Runtime/BB_Diagnostic_Services_Applications.md

```diff
@@ -0,0 +1,81 @@
+
+# Diagnostic Services Applications
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Runtime
+
+## Layer
+<!-- 1, 2a, 2b, 3 -->
+MWLayer
+
+## Known Implementation
+
```

Comment:
Eclipse SDV will soon have a project that is relevant for this BB (formerly called kanto.auto, name will change)

Reply:
Agreed. Will be added when available.